src/src/components/spotify/SpotifyLogo.jsx

```
@@ -0,0 +1,12 @@
+const SpotifyLogo = () => {
+  return (
+    <svg viewBox="0 0 147 147" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)